content/en/docs/Tasks/install-operator-with-olm.md

```diff
@@ -6,6 +6,21 @@ description: >
   [Once you've made your operator available in a catalog](/docs/tasks/make-operator-part-of-catalog/), or you've chosen an operator from the [list of operators available to be installed in the cluster](/docs/tasks/list-operators-available-to-install/), you can install your operator by creating a [`Subscription`](/docs/concepts/customresourcedefinitions/subscription/) to a specific [channel](/docs/concepts/glossary/#channel). 
 ---
 
+Before installing into a namespace, you will need to create an [operator group](/docs/concepts/crds/operatorgroup) in that namespace. The operator group must provide the namespaces that match the operator that you trying to install `installModes`.
```

Comment:
```suggestion
Before installing an Operator in a namespace, you will need to create an [OperatorGroup](/docs/concepts/crds/operatorgroup) in that namespace. The operator group must provide the namespaces that match the operator that you trying to install `installModes`.
```

Reply:
`The operator group must provide the namespaces that match the operator that you trying to install `installModes`.` 

This sentence is confusing to me. Maybe it'll be clear if you elaborated on the part about "match"-ing the operator?